app/src/main/java/com/example/myapplicationwork/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.myapplicationwork
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
```

Comment:
please remove unused comments/fields/methods

Reply:
you can use instead of override `onCreateView`
like so class `FragmentMoviesDetails : Fragment(R.layout. fragment_movies_details)`